Provis.WebApi/Controllers/WorkspaceController.cs

```diff
@@ -51,6 +53,7 @@ public async Task<IActionResult> AcceptInviteUserAsync(int id)
 
         [Authorize]
         [HttpPost]
+        [WorkspaceRoles(new int[] { (int)WorkSpaceRoles.OwnerId, (int)WorkSpaceRoles.ManagerId })]
```

Comment:
Let's set enum values as attribute values

Reply:
I've changed it